src/lib/y2storage/disk_analyzer.rb

```diff
@@ -48,24 +48,6 @@ module Y2Storage
   class DiskAnalyzer
     include Yast::Logger
 
-    LINUX_PARTITION_IDS =
-      [
-        PartitionId::LINUX,
-        PartitionId::SWAP,
-        PartitionId::LVM,
-        PartitionId::RAID
-      ]
-
-    WINDOWS_PARTITION_IDS =
-      [
-        PartitionId::NTFS,
-        PartitionId::DOS32,
-        PartitionId::DOS16,
-        PartitionId::DOS12,
-        PartitionId::WINDOWS_BASIC_DATA,
-        PartitionId::MICROSOFT_RESERVED
-      ]
-
```

Comment:
Because the logic for `candidate_disks` remains in DiskAnalyzer. I moved the other two constants because the logic was moved to PartitionId.